single-audit/Pipfile

```diff
@@ -15,6 +15,7 @@ django-localflavor = "*"
 designstandards = {editable = true,git = "https://github.com/department-of-veterans-affairs/django-designstandards.git"}
 pandas = "*"
 jupyter = "*"
+pytz = "*"
```

Comment:
I don't see `import pytz` anywhere in the diff'ed files, so is this library being used?

Reply:
It's required by `timezone`, which is being imported in line 2 of `models.py`.

Reply:
Gotcha — took a look at the docs. They mention "Time zone support uses pytz, which is installed when you install Django." 

https://docs.djangoproject.com/en/2.2/topics/i18n/timezones/

`pytz` is already included in our `Pipfile.lock` on `master`, so I'd assume we can remove this line safely. 

https://github.com/18F/federal-grant-reporting/blob/master/single-audit/Pipfile.lock#L408

Reply:
Makes sense — thanks!

Reply:
Ironically, when I went to rebase on master, git rebase choked on 1) this commit and 2) the commit that reverted it. Force-pushed.